config/config.exs

```diff
@@ -15,9 +15,16 @@ config :screenplay, ScreenplayWeb.Endpoint,
   pubsub_server: Screenplay.PubSub,
   live_view: [signing_salt: "vSiyKz7D"]
 
+sftp_client_module =
+  case System.get_env("SFTP_SERVER") do
```

Comment:
I didn't realize we were setting the real/fake client module in this way. If we move this to `config`, won't it no longer work as expected, since it is now reading the environment variable at build time instead of once it starts up on the server? Is there a reason to use an env var for this rather than e.g. setting it to use the fake client in `test.exs` and the real one otherwise?

Reply:
Not 100% on the OG reason, but I think it's because prod is the only environment that uses the real `SFTPClient` module. Dev and dev-green use the fake one (unless we decide to hook up the logic in those environments to use the new test folder we write to in the new job). I feel like needing a separate variable for that is a bit overkill considering we have access to `ENVIRONMENT_NAME`. Should I move this back to `runtime.exs` and go back to using a helper function in `sftp.ex` and `takeover_tool_testing_job.ex` to get the correct client module?

Reply:
For the moment, to get this to a working state since it's already on the main branch, that might be the best approach.

For the future, I like your idea of writing to the test directory in the other environments. The more closely aligned the `dev` environments are with the code that runs on `prod`, the better, otherwise we can't have as much confidence that if something works in one place it will work in the other.